omnigibson/examples/object_states/folded_unfolded_state_demo.py

```diff
@@ -1,5 +1,5 @@
 from omnigibson.utils.constants import PrimType
-from omnigibson.object_states import Folded
+from omnigibson.object_states import Folded, Unfolded
```

Comment:
high level question: what was the reason for having an explicit separate state rather than having `Folded` be tertiary?